### PR TITLE
Add definition for root span

### PR DESCRIPTION
**Pull request by wantsui**
### What does this PR do?

Explains what the root span of a trace is.

### Motivation

This definition is necessary to understand how to use other APM features.

### Preview

https://docs-staging.datadoghq.com/wantsui/rootspan-definition/tracing/visualization

### Additional Notes

Should be reviewed by @andrewardito

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/visualization/_index.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ The APM UI provides many tools to troubleshoot application performance and corre
 | [Trace](#trace)                 | A trace is used to track the time spent by an application processing a request and the status of this request. Each trace consists of one or more spans.                                                             |
 | [Span](#spans)                  | A span represents a logical unit of work in a distributed system for a given time period. Multiple spans construct a trace.                                                                                          |
 | [Top Level Span](#top-level-span) | A span is a top level span when it is the entrypoint method for a request to a service.  You can visualize this within Datadog APM when the color of the immediate parent on a flamegraph is a different color.                                                                                            |
+| [Root span](#root-span) | The first span generated by a request and, as a result, does not have a parent                      |
```
Comment (cswatt) [inline content/en/tracing/visualization/_index.md:34]:
```suggestion
| [Root span](#root-span) | The first span generated by a request and, as a result, does not have a parent.                      |
```

File: content/en/tracing/visualization/_index.md [hunk 2]
```diff
@@ -98,6 +99,10 @@ For the example below, the top level spans are:
 
 {{< img src="tracing/visualization/toplevelspans.png" alt="span" >}}
 
+## Root span
+
+The very first span generated by a request and, as a result, does not have a parent. You can visualize this on the flamegraph by finding the span on the top of the flamegraph that initiates the request.
```
Comment (cswatt) [inline content/en/tracing/visualization/_index.md:104]:
```suggestion
The root span is the very first span generated by a request and, as a result, does not have a parent. You can visualize this on the flamegraph by finding the span on the top of the flamegraph that initiates the request.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
small grammar/style stuff

**Comment by wantsui** [comment]
I'm going to close this PR for now since the definitions will be revisited at a later time.

**Pull request closed without merging**